cfbs/commands.py

```diff
@@ -521,7 +521,7 @@ def _clean_unused_modules(config=None):
         return 0
 
     def _someone_needs_me(this) -> bool:
-        if "added_by" not in this or this["added_by"] == "cfbs add":
+        if "added_by" not in this or is_module_added_manually(this["added_by"]):
```

Comment:
Operator precedence can be a bit confusing sometimes. Hence, adding parentheses around could help the readability
```suggestion
        if ("added_by" not in this) or is_module_added_manually(this["added_by"]):
```
But feel free to dismiss, if you disagree.